ravenpy/models/base.py

```diff
@@ -323,16 +286,16 @@ def run(self, ts, overwrite=False, **kwds):
         # Case for potentially parallel parameters
         pdict = {}
         for p in self._parallel_parameters:
-            a = kwds.pop(p, None)
-
-            if (
-                a is not None
-                and p == "params"
-                and not isinstance(a, self.__class__.Params)
-            ):
-                pdict[p] = np.atleast_2d(a)
+            val = kwds.pop(p, None)
+            if val is not None and p == "params":
+                # Special case where we have `Params(..)` or `[Params(), ..]`
+                lval = [val] if not is_sequence(val) else val
```

Comment:
Wondering if this will break passing a string of parameters, e.g. 
`1.2, 35, 4.56` which is understood by NamedTuple objects, and used by WPS processes to instantiate params.

Reply:
Should be ok.